packages/orm/src/newTestInstance.ts

```diff
@@ -181,16 +187,15 @@ function resolveFactoryOpt<T extends Entity>(
       }
       // Otherwise fall though to making a new entity via the factory
     }
-    // Find the opposite side, to see if it's an o2o or o2m pointing back at us
-    const otherField = field.otherMetadata().fields[field.otherFieldName];
+    if (maybeEntity === undefined) {
+      // If this is image.author (m2o) but the other-side is a o2o, pass null instead of []
+      maybeEntity = (field.otherMetadata().fields[field.otherFieldName].kind === "o2o" ? null : []) as any;
+    }
```

Comment:
```suggestion
    if (maybeEntity === undefined) {
      // If this is image.author (m2o) but the other-side is a o2o, pass null instead of []
      maybeEntity = (field.otherMetadata().fields[field.otherFieldName].kind === "o2o" ? null : []) as any;
    }
```
```suggestion
    // If this is image.author (m2o) but the other-side is a o2o, pass null instead of []
    maybeEntity ??= (field.otherMetadata().fields[field.otherFieldName].kind === "o2o" ? null : []) as any;
```

Reply:
Ah nice, updated